handsontable.d.ts

```diff
@@ -1743,9 +1743,19 @@ declare namespace Handsontable {
       classNames: string[];
     }
 
+    interface CellCoords {
+      col: number;
+      row: number;
+    }
+
+    interface CellRange {
+      from: CellCoords;
+      to: CellCoords;
+    }
```

Comment:
We already have CellCoords and CellRange on `wot`. Should we use them instead creating new ones @budnix ?

Reply:
Yep. There is a reason that the `CellCoords` and `CellRange` interfaces are inside the Walkontable namespace (`wot`). The Walkontable does not provide its own definition so we wanted to emphasize the interface owner.

Reply:
`wot.CellRange` has `highlight: CellCoords` which we don't want so I made new ones, how do you reckon this should be solved then?

Reply:
I dunno, @budnix ?